lab_4_retrieval_w_clustering/main.py

```diff
@@ -154,6 +220,13 @@ def get_raw_documents(self, indices: tuple[int, ...] | None = None) -> Corpus:
         Returns:
             Corpus: List of documents.
         """
+        if not indices:
+            return self.__documents
+        unique = []
```

Comment:
set?

Reply:
no cause i can't change the order of elements inside indices